server/controllers/EmailController.js

```
@@ -270,6 +283,94 @@ const EmailController = {
     }
   },
 
+  routeTeamEmail: async function(req, res, next){
+    const { to, html, from, text, subject } = req.body;
+
+    let senderEmail;
+    if( from && typeof from === 'string' && from.length ){
+      const emailStart = from.indexOf('<') + 1;
+      const emailEnd = from.indexOf('>');
+      senderEmail = from.trim().toLowerCase().slice( emailStart, emailEnd );
+    }
```

What if the if-condition isn't true? You should raise an exception.